python_modules/dagster/dagster/core/telemetry.py

```diff
@@ -128,10 +128,26 @@ def get_python_version():
 
 
 class TelemetryEntry(
-    namedtuple(
-        "TelemetryEntry",
-        "action client_time elapsed_time event_id instance_id pipeline_name_hash "
-        "num_pipelines_in_repo repo_hash python_version metadata version dagster_version os_desc os_platform",
+    NamedTuple(
+        "_TelemetryEntry",
+        [
+            ("action", str),
+            ("client_time", str),
+            ("event_id", str),
+            ("elapsed_time", str),
+            ("instance_id", str),
+            ("metadata", Dict[str, str]),
```

Comment:
I'm not familiar with the telemetry code, is `metadata` here different than it is in other contexts (i.e. this doesn't use the `EventMetadata` system)?

Reply:
Yea it's metadata specific to a specific type of action, not really related to execution in any meaningful way.